lib/mayaUsd/ufe/UsdTransform3dCommonAPI.h

```diff
@@ -85,13 +86,9 @@ class MAYAUSD_CORE_PUBLIC UsdTransform3dCommonAPIHandler : public Ufe::Transform
 
     // Ufe::Transform3dHandler overrides
     Ufe::Transform3d::Ptr transform3d(const Ufe::SceneItem::Ptr& item) const override;
-    Ufe::Transform3d::Ptr editTransform3d(
-        const Ufe::SceneItem::Ptr& item
-#if UFE_PREVIEW_VERSION_NUM >= 2030
+    Ufe::Transform3d::Ptr editTransform3d(const Ufe::SceneItem::Ptr& item UFE_V2(
         ,
-        const Ufe::EditTransform3dHint& hint
-#endif
-    ) const override;
+        const Ufe::EditTransform3dHint& hint)) const override;
```

Comment:
@kxl-adsk This is clang-format weirdness. It happened 4 times in my changes (all same kind of formatting). Is there anything we can do about this?

Reply:
Please, log it in our internal wiki page

Reply:
Done.